<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1475</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>播放列表</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_add">
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/new/prefix1/QIcon/images/316.bmp</normaloff>:/new/prefix1/QIcon/images/316.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_del">
           <property name="text">
            <string>删除</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/new/prefix1/QIcon/images/318.bmp</normaloff>:/new/prefix1/QIcon/images/318.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear">
           <property name="text">
            <string>清空</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/new/prefix1/QIcon/images/214.bmp</normaloff>:/new/prefix1/QIcon/images/214.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QVideoWidget" name="VideoWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>720</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>视频播放操作</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_pause">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/new/prefix1/QIcon/images/620.bmp</normaloff>:/new/prefix1/QIcon/images/620.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_play">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/new/prefix1/QIcon/images/622.bmp</normaloff>:/new/prefix1/QIcon/images/622.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stop">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/new/prefix1/QIcon/images/624.bmp</normaloff>:/new/prefix1/QIcon/images/624.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_last">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/new/prefix1/QIcon/images/616.bmp</normaloff>:/new/prefix1/QIcon/images/616.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_next">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/new/prefix1/QIcon/images/630.bmp</normaloff>:/new/prefix1/QIcon/images/630.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>538</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/new/prefix1/QIcon/images/mute.bmp</normaloff>:/new/prefix1/QIcon/images/mute.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_setSound">
            <property name="value">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>60</height>
          </size>
         </property>
         <property name="title">
          <string>播放进度</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_name">
            <property name="text">
             <string>文件名：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>428</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_pro">
            <property name="text">
             <string>进度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_pro">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>22</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_time">
            <property name="text">
             <string>时长</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
